testsuite/features/build_validation/add_common_repositories/alma9_minion_add_common_repositories.feature

```diff
@@ -2,7 +2,7 @@
 # Licensed under the terms of the MIT license.
 
 @alma9_minion
-Feature: Add the Alma 9 distribution custom repositories
+Feature: Add the Alma 9 distribution common repositories
   In order to use Alma 9 channels with Appstream modules
   As a SUSE Manager administrator
   I want to filter them out to remove the modules information
```

Comment:
I'm reviewing from my mobileapp, so maybe the code is in fact fine, but this line looks bad indented